app/assets/javascripts/geoblacklight/modules/relations.js.erb

```diff
@@ -0,0 +1,80 @@
+var setupDataRelations = function () {
+    $('#sidebar').append($('<div class="panel panel-default show-tools" id="relation_tools"><div class="panel-heading">Data Relations</div><div class="panel-body"><ul class="nav"><div class="ancestor_links"></div><div class="descendant_links"></div></ul></div></div>').hide().fadeIn(200));
+}
+
+var setupAncestorLinks = function (count) {
+    $('#relation_tools .nav').append('<b>Source Datasets (' + count + ')</b>');
+}
+
+var setupDescendantLinks = function (count) {
```

Comment:
I'm wondering if we can't provide a route that renders these and just inserts the HTML? (rather than providing JavaScript templates)